src/react/src/actions/dashboardModerationQueue.js

```diff
@@ -0,0 +1,165 @@
+import { createAction } from 'redux-act';
+import {
+    logErrorAndDispatchFailure,
+    downloadModerationEventsXLSX,
+} from '../util/util';
+
+export const startFetchingModerationEvents = createAction(
+    'START_FETCHING_MODERATION_EVENTS',
+);
+export const failFetchingModerationEvents = createAction(
+    'FAIL_FETCHING_MODERATION_EVENTS',
+);
+export const completeFetchingModerationEvents = createAction(
+    'COMPLETE_FETCHING_MODERATION_EVENTS',
+);
+export const startDownloadingModerationEvents = createAction(
+    'START_DOWNLOADING_MODERATION_EVENTS',
+);
+export const failDownloadingModerationEvents = createAction(
+    'FAIL_DOWNLOADING_MODERATION_EVENTS',
+);
+export const completeDownloadingModerationEvents = createAction(
+    'COMPLETE_DOWNLOADING_MODERATION_EVENTS',
+);
+
+const mockData = [
```

Comment:
Please add TODO to remove it when integration task will be implemented.

Reply:
Done.